aptos-move/block-executor/src/errors.rs

```diff
@@ -13,28 +14,29 @@ pub enum IntentionalFallbackToSequential {
     /// TODO: (short-mid term) relax the limitation, and (mid-long term) provide proper multi-versioning
     /// for code (like data) for the cache.
     ModulePathReadWrite,
-    /// We defensively check certain resource group related invariant violations.
-    ResourceGroupError(String),
+    /// We defensively check resource group serialization error in the commit phase.
+    /// TODO: should trigger invariant violation in the transaction itself.
```

Comment:
no if agg_v2 are ones we want to fix now - because we can't do that for now (that's why we are doing the skip-fallback next), but whenever we do the session/whatever project, we should address this?